Client/qtTeamTalk/common.h

```diff
@@ -66,7 +66,7 @@
 #endif
 
 #if defined(Q_OS_WINCE)
-#define _W(qstr) qstr.utf16()
+#define _W(qstr) qstr.utf8()

```

Comment:
How come this change? Windows CE is no longer supported

Reply:
Hmmmmm, a mistake that would not have been there